populate_dev_data_handler.go

```diff
@@ -68,6 +70,33 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
         },
     }
 
+    // Get the redirects, but don't follow them.
+    var errUseLastResponse = errors.New("hey... don't redirect")
```

Comment:
Can the error string be something more descriptive?

Reply:
Done.